db/tables.py

```diff
@@ -1,7 +1,9 @@
-from sqlalchemy import Column, String, Table, MetaData, func, select
+from sqlalchemy import (
+    Column, String, Table, MetaData, func, select, ForeignKey, literal, exists
+)
 
-from db import schemas
-from db.columns import init_mathesar_table_column_list_with_defaults
+from db import constants, schemas
+from db import columns as col
```

Comment:
I think it would be be better to import this as `columns` and change conflicting variable names, for clarity.